aladdin.sh

```diff
@@ -114,6 +114,10 @@ function _start_k3d() {
 }
 
 function check_or_start_k3d() {
+    if docker info | grep "Cgroup Version: 2" > /dev/null && [[ "$KUBERNETES_VERSION" == "1.19.7" ]]; then
+        echo "ERROR: Current version of k3d is not compatible with cgroups v2"
+        echo "ERROR: If using Docker Desktop please downgrade to v4.2.0"
+    fi
```

Comment:
I didn't add an `exit 1` in here in case it's a false positive (not sure if this is an issue in linux/windows as well), mostly just wanted to let the user know it might be an issue